handler/core/src/test/java/io/knotx/fragments/handler/action/CircuitBreakerActionTest.java

```diff
@@ -117,7 +133,8 @@ void expectTimeout(VertxTestContext testContext, Vertx vertx) throws Throwable {
                 l ->
                     Future.succeededFuture(
                         new FragmentResult(fragmentContext.getFragment(), SUCCESS_TRANSITION)
-                    ).setHandler(resultHandler)));
+                    ).setHandler(resultHandler)),
```

Comment:
Do we test timeouts?

I can not see tests that validates CircuitBreakerAction contract like:
- number of retries
- more doActionLog entries